install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -21,4 +21,6 @@ include::modules/ossm-deploying-bookinfo-application.adoc[leveloffset=+2]
 include::modules/ossm-about-accessing-bookinfo-application-using-gateway.adoc[leveloffset=+2]
 include::modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc[leveloffset=+2]
 include::modules/ossm-accessing-bookinfo-application-using-gateway-api.adoc[leveloffset=+2]
-include::modules/ossm-customizing-istio-configuration.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/ossm-customizing-istio-configuration.adoc[leveloffset=+1]
+//adding ossm-config-openshift-monitoring 08/13/2024 for the sake of adding it to an existing assembly. Thought to the IA of configurations/addons/integrations is needed.
```

Comment:
```suggestion
// adding ossm-config-openshift-monitoring 08/13/2024 for the sake of adding it to an existing assembly. Thought to the IA of configurations/addons/integrations is needed.
```
from the build history this looks like it might be causing issues too because without the space maybe it's not registering it as a comment?